roadmap/initiatives/mcp_integration.md

```
@@ -195,13 +195,13 @@ mcp_server.start()
 - None for Phase 1 (can start immediately)
 - Phase 2 depends on Phase 1 completion
 ## Risks
-**Risk 1: MCP Protocol Changes**
+#### Risk 1: MCP Protocol Changes
```

markdownlint-cli2 (0.18.1)

Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)